src/modules/facility-loan/dto/create-facility-loan-request.dto.ts

```diff
@@ -70,4 +70,16 @@ export class CreateFacilityLoanRequestItem {
     description: 'The facility expiry date.',
   })
   readonly expiryDate: DateOnlyString;
+
+  @ValidatedDateOnlyApiProperty({
+    description: 'The next payment due date of the repayment schedule.',
+  })
+  readonly nextDueDate: DateOnlyString;
+
+  @ValidatedStringApiProperty({
+    description: 'The frequency at which loan bills should be generated.',
+    length: 1,
+    example: 'C',
```

Comment:
Can this be ENUM?

Reply:
I did a GET request to ACBS to `/Lists/LoanBillingFrequencyType` to see the options but it looks like they can be edited in ACBS so decided that it was ok to just validate the length